pkg/core/cmdutil/selfupdate.go

```diff
@@ -0,0 +1,60 @@
+package cmdutil
+
+import (
+	"time"
+
+	"github.com/blang/semver"
+	"github.com/redhat-developer/app-services-cli/internal/build"
+	"github.com/redhat-developer/app-services-cli/pkg/core/config"
+	"github.com/redhat-developer/app-services-cli/pkg/core/logging"
+	"github.com/rhysd/go-github-selfupdate/selfupdate"
+)
+
+func DoSelfUpdate(logger logging.Logger) (bool, error) {
+	version := build.Version
+	if build.IsDevBuild() {
+		// TODO uncomment later
+		//return false, nil
+		version = "0.0.0"
+	}
+
+	v := semver.MustParse(version)
+	latest, err := selfupdate.UpdateSelf(v, build.RepositoryOwner+"/"+build.RepositoryName)
```

Comment:
I think we should ask user before performing update instead of doing this automatically.

Additionally we need to show some progress/successful mesage.

In some cases update would not be able to be performed unless `sudo rhoas` is executed.